weblab/templates/base.html

```diff
@@ -1,20 +1,8 @@
-{% load staticfiles %}
-
-<!--
-
-    ___     / __      ___      ___    __  ___  ___
-  //   ) ) //   ) ) //   ) ) ((   ) )  / /   //___) )
- //       //   / / //   / /   \ \     / /   //
-((____   //   / / ((___( ( //   ) )  / /   ((____
-
-
--->
-
 <!DOCTYPE html>
 <html>
   <head>
     <title>
-      {% block title %}{% endblock %}Functional Curation @ Chaste - Cardiac Electrophysiology
+      {% block title %}{% endblock %}Web Lab - Cardiac Electrophysiology
```

Comment:
Are we using this name to indicate the other future uses? I've been calling it the "Cardiac Electrophysiology Web Lab" :-)

Reply:
That was what I was thinking; well that and making a smaller edit :)